ansible_collections/arista/avd/tests/integration/targets/yaml_templates_to_facts/tests/test_with_template_output_false.yml

```diff
@@ -0,0 +1,21 @@
+- name: Test with template_output false
+  ignore_errors: no
+  register: result
+  yaml_templates_to_facts:
+    template_output: false
+    templates:
+      - template: "templates/inline_jinja.j2"
+
+- assert:
+    that:
+      - result is success
+      - result.changed == true
+      - result.ansible_facts is defined
+      - result.ansible_facts.key_1 is defined
+      - result.ansible_facts.key_1 == "value_1"
+      - result.ansible_facts.key_2 is defined
+      - result.ansible_facts.key_2[0] == '{'
+      - result.ansible_facts.key_2[1] == '{'
+      - result.ansible_facts.key_2[2:9] == ' key_1 '
+      - result.ansible_facts.key_2[9] == '}'
+      - result.ansible_facts.key_2[10] == '}'
```

Comment:
Is there any specific reason to add assert for characters instead of for whole string?. Can we add assert like ansible_facts.key_2 == '{{  key_1 }}'

Reply:
Ansible automatically tried to render any value that looks like jinja, so the {{ will trigger an error. Here I specifically want to test if the content is the string `{{ key_1 }}` and _not_ the value of key_1. I tried escaping etc, but couldn't find a way.

Reply:
Got it. Thanks